README.md

```diff
@@ -26,3 +26,29 @@ When making changes to the Cookie Cutter, keep the following in mind:
 * update pins in requirements/*.in files but *don't* commit the compiled requirements.txt
   files to the repo.
 * update to latest Python supported by Django. For Django 4.1 this is 3.8, 3.9, and 3.10.
+
+## Python Libraries
+
+As an optionated template, we choose as dependencies some Python libraries and frameworks that
+are configured and made available to be used during the project development. Some of them
+will be installed only if requested when answering the questions in the project creation.
```

Comment:
It isn't necessary to restate that this is an opinionated template.
```suggestion
This template includes the Python libraries listed below. Some of them
will be installed only if requested when answering the questions in the project creation.
```
Maybe we should point out which libraries are tied to which questions eg. celery, django-celery-beat, etc will only be installed if you request it during project creation.